helm/kubernetes-nginx-ingress-controller-chart/templates/controller-hpa.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ .Values.controller.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.controller.name }}
+    giantswarm.io/service-type: "managed"
+    k8s-app: {{ .Values.controller.k8sAppLabel }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ .Values.controller.name }}
+  minReplicas: 2
```

Comment:
I'm thinking we might want to make this configurable, and even if not I'd suggest changing it it to templated value with a default in values.yaml. same with max, and the 2 utilization values below (I'd do them separately)

use case for making min user configurable: some customers that run webscale loads know they never wanna go below x amount of ICs at least in certain times of year if not always.